transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -1163,6 +1193,17 @@ public Builder ticker(Ticker ticker) {
         public EmbeddedChannel build() {
             return new EmbeddedChannel(this);
         }
+
+        public static EmbeddedChannel of(ChannelId channelId, boolean hasDisconnect, ChannelConfig config,
```

Comment:
This looks strange honestly...

Reply:
Do you mean the method name or the list of params?

Reply:
Its strange to have this part of the builder.. why not just make it a constructor of `EmbeddedChannel`

Reply:
I initially did that, but later I thought it might bring back compatibility issues. If you're sure it won't bring any trouble, I'll replace it with a constructor.

Reply:
You don't need the constructor